lib/middleware/middleware.ts

```diff
@@ -10,9 +10,13 @@ import { NextApiRequest, NextApiResponse } from "next";
  */
 export const middleware = (
   middlewareArray: Array<MiddlewareRequest>,
-  handler: (req: NextApiRequest, res: NextApiResponse, props: MiddlewareProps) => Promise<void>
+  handler: (
+    req: NextApiRequest,
+    res: NextApiResponse,
+    props: MiddlewareProps
+  ) => Promise<void | NodeJS.Timeout>
 ) => {
-  return async (req: NextApiRequest, res: NextApiResponse): Promise<void> => {
+  return async (req: NextApiRequest, res: NextApiResponse): Promise<void | NodeJS.Timeout> => {
```

Comment:
I know the reason why you changed the signature of the `middleware` function but I am wondering if it is relevant because the `submit` function in `submit.ts` does not look like it could return such `Timeout` object.

That code was aded by @Moro-Code through https://github.com/cds-snc/platform-forms-client/pull/416/files#diff-09973ff12369d6c2b43bd018d2b6f4c9869ec4da7d37f1d7d1efe840dbd9abecR39 . Do you remember why @Moro-Code ?

We might be able to refactor it so that we just remove the `Timeout` type in both places?

Reply:
Gotcha!

Reply:
Gotcha!